src/index.html

```diff
@@ -9,7 +9,66 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
+<form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+  <fieldset class="mb20">
+    <legend>Personal information</legend>
+    <label class="mb10">
+      Surname:<input type="text" name="surname" autocomplete="off"></label>
+    <label class="mb10">
+      Name:<input type="text" name="name" autocomplete="off"></label>
+    <label class="mb10">
+      How old are You?<input type="number" name="age" min="12" max="100" value="12"></label>
```

Comment:
```suggestion
    <label class="mb10">
      How old are You?<input type="number" name="age" min="12" max="100" value="12">
    </label>
```
Закрывающий тег должен быть на той же строке, что и открывающий